src/search.js

```diff
@@ -37,7 +37,8 @@ const search = ({
     ? getLeavingNow({ baseResults, hemisphere })
     : baseResults;
   console.log(sort);
-  return sortBy(sort)(resultsToReturn);
+  const sortedResults = sortBy(sort)(resultsToReturn);
+  return sort === "value" ? R.reverse(sortedResults) : sortedResults;
```

Comment:
What if we just add direction to `sortBy` so that we can also reverse the direction if we want to?

Reply:
Yeah that sounds like an ascending/descending toggle which I thought was out of the scope of this specifically though needed. The only reason I added this is because it seems like a value sort that is always ascending is terrible UX. So this was kind of a temporary patch until that functionality exists.

You want me to add it to this pr?

Reply:
Oh no that makes sense. Let's get this in and then I'll add an issue for the asc/dsc sort